docs/source/overview.md

```diff
@@ -1,21 +1,28 @@
 # Overview
 
-This documentation will refer to two roles that infrastructure providers can play:
+(Livepeer)[https://livepeer.org] is a live video streaming network protocol that is fully decentralized, highly scalable, crypto token incentivized, and results in a solution which is cheaper to an app developer or broadcaster than using traditional centralized live video solutions.
```

Comment:
Need fix the markdown link to livepeer.org i.e. [Livepeer](https://livepeer.org)